exampleSite/config.yaml

```diff
@@ -91,6 +91,12 @@ params:
   article_count: 3
 
   # brand and identity information
+  
+  # Shown on all pages. When logo is empty, it will show your site title, set above.
+  logo: ""
+  # White logo, or any alternate color of logo. Only shown on home page if set.
+  logo_white: ""
+  
```

Comment:
Not sure why the logo section was changed. Please revert this change to keep the scope narrow and clear.

BTW, remind me to show you how to do git patch commits sometimes with `git add --patch`. It is a super handy way to avoid committing too many changes by mistake.

Reply:
Looks good! 👍🏻